### ISSUE TITLE
Request: Add method to EXPAND an already existing certificate

### ISSUE THREAD

**Issue by techdaddies-kevin** [acmesh-official/acme.sh#2743]
We make changes to our certs a few times per month. We frequently need to add or remove domains from the certificate, and with the current acme.sh script, this is very frustrating, because we must use the `issue` function and pass in all of our original parameters that we used when generating the certificate. It would be VERY useful if you added two new commands:

`acme.sh --add -d site.com -w /home/site/public_html -d site2.com --dns dns_cloudns` AND
`acme.sh --remove -d site.com -d site2.com`

Upon running these commands, acme.sh would generate the full certificate using the EXISTING configuration, and simply add or remove the parameters as needed.

I hope this can be added. It would save us a lot of frustration.

**Comment by Neilpang** [comment]
we support different validation methods for each domain.
For example,
domain A is using webroot to validate
domain B is using  dns_cf 

Things can become complicated to edit the domains.

Let's see how many users want this.

**Comment by techdaddies-kevin** [comment]
Yeah, and we use multiple validations for our certs, which is why this is complicated for us to manage. We basically have to keep track of the issue command that we last used, then when we need to add or remove a domain from that cert, we copy the last used issue command, make our changes, and run it. It'd be so much easier for us to maintain if we could just issue a command to modify the existing cert. At the VERY least, if we just had a command that was like `acme.sh --show-params cert.com`, that would output the exact CLI command that was used to generate the cert, that would be helpful, so that we wouldn't have to manually maintain our copy of the command.

**Comment by amette** [comment]
As a workaround I edit the certificate config file directly and add the needed domain to the `LeAlt` option, then run `acme.sh --renew --force`.

**Comment by techdaddies-kevin** [comment]
We do the same thing, but we also have to update LE_Webroot manually when we do it. We have also had this cause acme to fail to add some domains (Never did figure that out).

**Comment by neilpang** [comment]
A built-in `--add`/`--remove` will not happen: each domain records its own validation method, so editing the set in place is exactly as complex as re-stating it, and a partial command that guesses the rest has too many failure modes. Two supported paths: (1) re-run the full `--issue` command with the updated `-d`/`-w`/`--dns` list -- keep it in a small script under version control so 'the last used command' is never lost; (2) the workaround above: edit `Le_Alt` (and `Le_Webroot` if validation differs) in the domain conf, then `--renew --force`. Closing.
